spec.bs

```diff
@@ -2027,6 +2115,27 @@ The network revocation mechanism requires the following monkeypatches to the [[F
 
   Add "[=must be blocked due to a revoked partition nonce=]" to the conditions after
   "should request be blocked by Content Security Policy".
+
+  Issue: This needs to be passed in both the fenced frame nonce as well as the iframe credentialless
+  nonce, if it exists.
+  (<a href="https://github.com/WICG/fenced-frame/issues/191">WICG/fenced-frame#191</a>)
+</div>
+
+The network revocation mechanism requires the following monkeypatches to the [[HTML]] Standard.
+
+<div algorithm=network-status-document-abort>
+  Add a new step to the [=abort a document=] algorithm after step 4 that says:
+
+  5. [=Recalculate the untrusted network status of all fenced frame descendants=] given <var
```

Comment:
I'm not entirely sure why this algorithm was modified to only be allowed to run in parallel. There's nothing that will block the main thread, so this should be okay to run both in parallel or in the document event loop. Unless you have any objections or there's some piece I'm missing, I'm going to remove that line.

This is needed to account for the following corner case:
1. A parent FF embeds a child FF.
2. The child FF disables its network.
3. The parent FF initiates a navigation of the child to a new config. This will result in a successful navigation and the network status reverting back to full untrusted access.
4. The parent FF immediately after calls `disableUntrustedNetwork()`. The in-progress navigation will be aborted. (Note this behavior is not spec'd yet, but there is [an issue logged for it](https://github.com/WICG/fenced-frame/issues/173))

If we don't recalculate the network statuses after the child FF's navigation is aborted, the promise will get stuck thinking some child frame still has network access.